star_tides/tests/fixtures/basic_user.py

```diff
@@ -0,0 +1,43 @@
+''' star_tides.tests.fixtures.basic_user
+'''
+import pytest
+import bcrypt
+from star_tides.services.mongo.models.user_model import User
+
+
+@pytest.fixture()
+def basic_user():
+    ''' Basic fixture for a user.
+    '''
+
+    created_users = []
```

Comment:
Nit: add type annotations in this file?

`created_users: List[User] = []` etc.

Reply:
In this specific instance, I don't think it would be that helpful here because the `created_users` variable is internal to this fixture and is just used to maintain the state of what the factory has built. 

I'm not opposed to this in the application though!